docs/en/writing-running-appium/touch-actions.md

```diff
@@ -13,6 +13,14 @@ These APIs allow you to build up arbitrary gestures with multiple actuators.
 Please see the Appium client docs for your language in order to find examples
 of using this API.
 
+**Note for W3C actions**
+
+[W3C actions](https://www.w3.org/TR/webdriver1/#actions) is also parcially available in some drivers such as XCUITest, UIA2 and Espresso.
+We've implemented W3C actions as possible, but OSs has limitations to behave full W3C compatible actions.
+e.g. WDA cannot handle zero wait [PR](https://github.com/appium/appium-xcuitest-driver/pull/753). Additional actions in [espresso](https://github.com/appium/appium-espresso-driver/issues/272).
```

Comment:
I would take out `Additional actions in [espresso](https://github.com/appium/appium-espresso-driver/issues/272).` since this isn't a limitation but rather an enhancement.